test/start-operator.sh

```diff
@@ -30,6 +30,8 @@ EOF
   ${SSH} "cat >'$tmpdir/kustomization.yaml'" <<EOF
 resources:
 - operator.yaml
+commonLabels:
+  pmem-csi.intel.com/deployment: ${TEST_OPERATOR_DEPLOYMENT:-operator}
```

Comment:
can you please add a comment about this new `TEST_OPERATOR_DEPLOYMENT` test environment variable how and when should be used.

Reply:
You didn't document `TEST_OPERATOR_NAMESPACE` either and that made sense to me because both variables are obvious. But I can explain both, of course.

Reply:
Done.